src/frontend/efiling-frontend/src/components/page/upload/Upload.js

```diff
@@ -1,22 +1,108 @@
 /* eslint-disable react/jsx-props-no-spreading */
 import React, { useState } from "react";
 import PropTypes from "prop-types";
-import { useDropzone } from "react-dropzone";
-import { Sidecard, Button } from "shared-components";
+import Dropzone from "react-dropzone";
+import { MdDescription, MdDeleteForever } from "react-icons/md";
+import {
+  Sidecard,
+  Button,
+  DisplayBox,
+  Table,
+  Dropdown,
+  Radio,
+} from "shared-components";
 import { getSidecardData } from "../../../modules/sidecardData";
 import { propTypes } from "../../../types/propTypes";
 
 import "./Upload.css";
 import PackageConfirmation from "../package-confirmation/PackageConfirmation";
 
+const items = [
+  "Select document description",
+  "Affidavit",
+  "Affidavit of Attempted Service",
+  "Case Conference Brief",
+];
+
+const generateFileJSX = (fileName) => {
+  return (
+    <div className="center-alignment fill-space">
+      <div style={{ color: "rgb(252, 186, 25)" }}>
+        <MdDescription size={32} />
+      </div>
+      <span className="file-href minor-margin-left">{fileName}</span>
+      <MdDeleteForever className="minor-margin-left" size={32} />
+    </div>
+  );
+};
+
+const generateRadioButtonJSX = (fileName, type) => {
+  return (
+    <div className="table-value">
+      <div className="minor-margin-right">
+        <Radio
+          id={`no-${type}-${fileName}`}
+          name={`${type}-${fileName}`}
+          label="No"
+          onSelect={(val) => console.log(val)}
```

Comment:
untested lines here I think

Reply:
yes as mentioned, all the onclick right now are just dummy console logs so they dont have tests associated with them causing the -0.3%
once backend changes in this can be addressed :)